libs/angular-utils/src/lib/components/portal-page/portal-page.component.ts

```diff
@@ -1,51 +1,30 @@
-import { Component, Inject, Input, LOCALE_ID, OnInit, Optional } from '@angular/core'
-import { HAS_PERMISSION_CHECKER, HasPermissionChecker } from '../../utils/has-permission-checker'
+import { Component, Input, OnInit } from '@angular/core'
 import { AppStateService } from '@onecx/angular-integration-interface'
-import { UserService } from '@onecx/angular-integration-interface'
 import { CommonModule } from '@angular/common'
 import { TranslateModule } from '@ngx-translate/core'
-import { TRANSLATION_PATH } from '../../utils/create-translate-loader.utils'
+import { PermissionService } from '../../services/permission.service'
+import { of } from 'rxjs'
 
 @Component({
   selector: 'ocx-portal-page',
   templateUrl: './portal-page.component.html',
   styleUrls: ['./portal-page.component.scss'],
   standalone: true,
   imports: [CommonModule, TranslateModule],
-  providers: [
-    {
-      provide: LOCALE_ID,
-      useFactory: (userService: UserService) => {
-        return userService.lang$.getValue()
-      },
-      deps: [UserService],
-    },
-    {
-      provide: TRANSLATION_PATH,
-      useValue: './onecx-angular-utils/assets/i18n/',
-      multi: true,
-    },
-  ],
 })
 export class PortalPageComponent implements OnInit {
   @Input() permission = ''
   @Input() helpArticleId = ''
   @Input() pageName = ''
   @Input() applicationId = ''
-
+  private defaultPermissionObs = of(true)
```

Comment:
```suggestion
  private trueObservable = of(true)
```

to have the same as in the tests